lms/templates/shoppingcart/shopping_cart.html

```diff
@@ -383,7 +383,7 @@ <h3 class="course-title-info" id="course-title">
       var typeChanged = false;
       var prevTotal = $('#total-amount').data('amount');
       var newTotal = getNewTotal(prevQty, newQty, unit_cost, prevTotal);
-      $('#total-amount').html('$'+newTotal.toFixed(2)+' USD');
+      $('#total-amount').html('${currency_symbol}'+newTotal.toFixed(2)+' ${currency.upper()}');
```

Comment:
Please add test to make sure this doesn't blow up if currency is null

(comment by @edx/ecommerce)